devenv/setup.sh

```diff
@@ -84,6 +112,8 @@ main() {
 		bulkDashboard
 	elif [[ $cmd == "bulk-folders" ]]; then
 		bulkFolders "$arg1"
+	elif [[ $cmd == "undev" ]]; then
```

Comment:
I think we should update the usage function as well :)

Reply:
You are adding a new command to the script, but did not mention that command is available in the `Usage` function a few lines above.